cermod/tripod/tripodMotionControl.cpp

```diff
@@ -1208,23 +1205,21 @@ bool tripodMotionControl::positionMoveRaw(const double *refs)
             yError() << "Requested position is not reachable";
         }
     }
-    _mutex.post();
 
     compute_speeds(_robotRef_positions, _lastRobot_encoders);
     // all joints may need to move in order to achieve the new requested position
     // even if only one user virtual joint has got new reference.
     ret &= _device.pos->setRefSpeeds(_njoints, _axisMap, _robotRef_speeds.data());
     ret &= _device.pos->positionMove(_njoints, _axisMap,_robotRef_positions.data());
 
-    _mutex.post();
```

Comment:
Hi @ale-git 
I've noticed that some code herein is protected by using `post` twice. I've used the lock-guard mechanism, hence one `post` is now gone. Was it intended for enforcing any sync? If so, feel free to amend this file.

Reply:
Hi @pattacini 
I'm sorry but I have no clue, Alberto Cardellino is the author I guess...

Reply:
Sorry @ale-git then 👍 
I think I will stick to the original version verbatim.

Reply:
There is no reason for having 2 post.
Like other vesion of the same positionMoveRaw, it was introduced in commit 85cfae4e49693e2f07c818fd5c00b28e3dae16df, then (should be) **_moved down_** in commit b53fa65346c118ef576123477dc70169fb495f8b. 
Other methods did it, that one didn't, I guess it was simply forgotten.

Reply:
Cool @barbalberto thanks!